executions/graphql-kotlin-dataloader-instrumentation/README.md

```diff
@@ -92,9 +89,9 @@ This way even if you are executing 2 separate operations you can still batch ope
 
 ### Usage in DataFetcher
 
-You can use the `DataFetchingEnvironment` which is passed to each
-`DataFetcher` and invoke the `getDataLoaderFromContext(dataLoaderName)` method which will access to the `KotlinDataLoaderRegistry`
-in the `GraphQLContext` and provide the `DataLoader` that you specified as `dataLoaderName` argument.
+To access to a `DataLoader` you can use the `DataFetchingEnvironment` which is passed to each
```

Comment:
To access a `DataLoader`, you can use the `DataFetchingEnvironment .getDataLoader(dataLoaderName: String)` extension function. Extension function will retrieve target data loader from the `KotlinDataLoaderRegistry` based on the specified `dataLoaderName`.

Reply:
i think you are referring to [getValueFromDataLoader](https://github.com/ExpediaGroup/graphql-kotlin/blob/master/servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/extensions/dataFetchingEnvironmentExtensions.kt#L27), the extension function is located in the server library.

we dont have a dependency on `graphql-java` on this specific library, so i think it would just be better to mention the `DataFetchingEnvironment.getDataLoader` method from official `graphql-java` library.

Reply:
👍